auth.js

```diff
@@ -0,0 +1,65 @@
+const { createSign } = require('crypto');
+
+const request = require('./request');
+
+const TOKEN_URL = 'https://oauth2.googleapis.com/token';
```

Comment:
This changed from the library we were using. I pulled this from the docs at https://developers.google.com/identity/protocols/oauth2/service-account#formingclaimset.